packages/shared/src/hooks/useRefreshToken.ts

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
Don't think we need this since hooks will only be called inside client components either way?